opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingDynamicFeature.java

```diff
@@ -197,6 +201,18 @@ public Builder withSkipPattern(String skipPattern) {
             return this;
         }
 
+        /**
+         * @param joinExistingActiveSpan If true, any active span on the current scope will be used as the parent span.
```

Comment:
`on the current thread`. `This can be used when combining with servlet instrumentation.`